static/org-settings.html

```diff
@@ -53,9 +53,6 @@ <h1 class="myOrg-heading">My Org</h1>
                             <div class="section-button"><a href="./test-data.html">Test Data</a></div>
                         </div>
                     </div>
-                    <div>
-                        <button type="button" id="logout-btn" class="btn">Logout</button>
-                    </div>
```

Comment:
Why don't we need this anymore?

Reply:
Since there is no org functionality, and the logout button does not do anything, it does not make sense to have it. Therefore, I removed it.